querybook/server/env.py

```diff
@@ -120,3 +120,6 @@ class QuerybookSettings(object):
 
     # Table Upload (Experimental)
     TABLE_MAX_UPLOAD_SIZE = get_env_config("TABLE_MAX_UPLOAD_SIZE")
+
+    # Event Logging
+    EVENT_LOGGER_NAME = get_env_config("EVENT_LOGGER_NAME")
```

Comment:
why not add a default value here? then you don't need DEFAULT_EVENT_LOGGER or the if logic